tracing/README.md

```diff
@@ -93,44 +89,81 @@ be invoked with the same syntax as the similarly-named macros from the `log`
 crate. Often, the process of converting a project to use `tracing` can begin
 with a simple drop-in replacement.
 
-Let's consider the `log` crate's yak-shaving example:
+Let's consider the `log` crate's yak-shaving
+[example](https://docs.rs/log/0.4.6/log/index.html#examples), tweeked to
+support tracing:
 
 ```rust
-#[macro_use]
-extern crate tracing;
-use tracing::field;
+// Add a use statement so we get tracing version of macros
+use tracing::{span, info, warn, Level};
+
+// Dummy impls to make the example compile
+#[derive(Debug)] pub struct Yak(String);
+impl Yak { fn shave(&mut self, _: u32) {} }
+fn find_a_razor() -> Result<u32, u32> { Ok(1) }
 
 pub fn shave_the_yak(yak: &mut Yak) {
-    // Create a new span for this invocation of `shave_the_yak`, annotated
-    // with  the yak being shaved as a *field* on the span.
-    span!("shave_the_yak", yak = field::debug(&yak)).in_scope(|| {
-        // Since the span is annotated with the yak, it is part of the context
-        // for everything happening inside the span. Therefore, we don't need
-        // to add it to the message for this event, as the `log` crate does.
-        info!(target: "yak_events", "Commencing yak shaving");
-
-        loop {
-            match find_a_razor() {
-                Ok(razor) => {
-                    // We can add the razor as a field rather than formatting it
-                    // as part of the message, allowing subscribers to consume it
-                    // in a more structured manner:
-                    info!({ razor = field::display(razor) }, "Razor located");
-                    yak.shave(razor);
-                    break;
-                }
-                Err(err) => {
-                    // However, we can also create events with formatted messages,
-                    // just as we would for log records.
-                    warn!("Unable to locate a razor: {}, retrying", err);
-                }
+    // Add a span and enter it to utilize scoped logging in tracing
```

Comment:
I think this comment should be reworded.

Alternatively, why not remove the span from this example, and show it in the next one?

Reply:
I like the second idea